src/features/home/events/events.tsx

```diff
@@ -6,53 +10,45 @@ import {
   Subtitle,
   Title,
 } from '@/app/components';
+import photo3 from '@/assets/photo-3.webp';
+import Image from '@/features/image';
+import { getActualDataList } from '@/utils';
 
-import './events.scss';
-
-const events: EventCardProps[] = [
-  {
-    eventType: 'Meetup',
-    title: 'Autoscaling strategies for ECS Fargate',
-    organizedBy: 'by AWS User Group',
-    organization: '3CITY / Trójmiasto',
-    date: '29 June 2023',
-    time: '18:00',
-    type: 'Offline',
-    address: 'al. Grunwaldzka 472B',
-    city: 'Gdańsk',
-    href: 'https://www.meetup.com/aws-user-group-3city/events/293689995/',
-  },
-  {
-    eventType: 'Meetup',
-    title: 'How Does the Ideal CI/CD Pipeline Look?',
-    organizedBy: 'Vilnius, Lithuania',
-    organization: 'Rolling Scopes Meetup',
-    date: '13 July 2023 2023',
-    time: '19:00',
-    type: 'Offline',
-    address: 'Vytenio Str., 18',
-    city: 'Vilnius',
-    href: 'https://www.meetup.com/the-rolling-scopes-lithuania/events/293907703/',
-  },
-];
+import styles from './events.module.scss';
+
+const cx = classNames.bind(styles);
+
+const displayedCardsQuantity = 2;
+
+const actualEvents = getActualDataList({
+  dataList: events as EventCardProps[],
```

Comment:
```suggestion
  dataList: events,
```

Reply:
This function is universal for all such lists - for example, events, courses, etc. / Therefore, I carry out typing at the stage of its use

Reply:
You've defined the variables type in the creation phase, you don't need to type cast

![image](https://github.com/rolling-scopes/site/assets/114234698/e4b66911-c25a-45a8-bb20-fdf146ccaa2c)
![image](https://github.com/rolling-scopes/site/assets/114234698/4dcd2aca-b598-4e7c-b4ba-ad2879672a8e)